src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1559,7 +1558,8 @@ namespace winrt::TerminalApp::implementation
             activeTab->ToggleBroadcastInput();
             args.Handled(true);
         }
-        // If the focused tab wasn't a TerminalTab, then leave handled=false
+        // TODO(DH)

```

Comment:
I marked this todo not in the scope of this PR, but because after Mike's rewrite _we never left handled=false_. There will always be a sender or focused tab.